packaging/ironbank/hardening_manifest.yaml.tmpl

```diff
@@ -13,7 +13,7 @@ tags:
 # Build args passed to Dockerfile ARGs
 args:
   BASE_IMAGE: "redhat/ubi/ubi8"
-  BASE_TAG: "8.6"
+  BASE_TAG: "8.8"
   ELASTIC_STACK: "{{ beat_version }}"
```

Comment:
As far as I see, https://github.com/elastic/apm-server/pull/8732 was not backported to 7.17 hence we keep using `beat_version`